plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -86,6 +86,7 @@
     private int maxConcurrentFileRenames = 20;
```

Comment:
Please shorten the commit message so it shows up as one line, and any more info can go in description.

Reply:
sg